CONTRIBUTING.md

```diff
@@ -490,3 +490,30 @@ We have a few procedures we follow. These are documented here:
   
   </p>
   </details>
+
+### Set up a Docker Builder
+
+- amd64, armv7 using local.
```

Comment:
Bit unclear what is being documented here. what are these bullet points?

An introductory sentence why one would need/want to set up a docker builder might be helpfull for other devs that did not read this PR.

Reply:
It is a Docker config that is for me to build the multi arch images. Probably for myself only, can be improved if anyone else need this later.